projects/packages/my-jetpack/_inc/data/utils/get-product-slugs-that-require-user-connection.ts

```diff
@@ -8,7 +8,7 @@ const getProductSlugsThatRequireUserConnection = ( products: {
 		.filter(
 			( { requiresUserConnection, status } ) =>
 				requiresUserConnection &&
-				( status === PRODUCT_STATUSES.ACTIVE || status === PRODUCT_STATUSES.ERROR )
+				( status === PRODUCT_STATUSES.ACTIVE || PRODUCT_STATUSES.USER_CONNECTION_ERROR )
```

Comment:
This was returning an empty array always because `PRODUCT_STATUSES.ERROR` no longer exists